src/components/functional_module/src/plugin_manager.cc

```diff
@@ -43,6 +43,38 @@ namespace functional_modules {
 
 CREATE_LOGGERPTR_GLOBAL(logger_, "PluginManager")
 
+namespace {
+const std::string ExtractMethodName(application_manager::MessagePtr msg) {
+  Json::Value value;
+  Json::Reader reader;
+  reader.parse(msg->json_message(), value);
+  
+  const char* kMethod = "method";
+  const char* kResult = "result";
+  const char* kError = "error";
+  const char* kData = "data";
+
+  if (value.isMember(kMethod)) {
+    return value.get(kMethod, "").asCString();
+    // Response from HMI
```

Comment:
@LuxoftAKutsan 
I would rather remove that. This method was just moved here from other place.